### ISSUE TITLE
Failing to get answers

### ISSUE THREAD

**Issue by smartvnm** [lagranges/people_also_ask#1]
`$ python test.py

ERROR: test_get_answer (__main__.TestGoogle)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/storage/emulated/0/script/test.py", line 20, in test_get_answer
    self.assertIsNotNone(answer["response"])
KeyError: 'response'

----------------------------------------------------------------------
Ran 2 tests in 3.953s

FAILED (errors=1)`

**Comment by lagranges** [comment]
Hi @smartvnm,

Thank you for reporting this issue,

I reran unittest, it works for me.
Can you tell me if it's sysematic or random error ?

**Comment by smartvnm** [comment]
Hi, I've been digging a bit on this. It appears that Google uses your IP address to give you local results (country-specific). What I didn't check was how your code scrapes Google, but by playing with other's scripts (they are getting the text based on tags) I found out that the page Google serves in my country is different than US Google page. I used a VPN service to bypass that and got good results.
I will retest later with your script behind a VPN, but most likely this is the problem.

L.E. Yup, that's the case

`$ python test.py
..
----------------------------------------------------------------------
Ran 2 tests in 5.292s

OK`
